khelo/lib/ui/flow/team/add_team_member/add_team_member_screen.dart

```diff
@@ -153,31 +185,34 @@ class _AddTeamMemberScreenState extends ConsumerState<AddTeamMemberScreen> {
     BuildContext context,
     AddTeamMemberState state,
   ) {
-    return AppTextField(
-      controller: state.searchController,
-      borderRadius: BorderRadius.circular(30),
-      contentPadding: const EdgeInsets.all(16),
-      borderType: AppTextFieldBorderType.outline,
-      onChanged: (value) => notifier.onSearchChanged(),
-      backgroundColor: context.colorScheme.containerLowOnSurface,
-      hintText: context.l10n.add_team_member_search_placeholder_text,
-      style: AppTextStyle.body2.copyWith(
-        color: context.colorScheme.textPrimary,
-      ),
-      hintStyle: AppTextStyle.subtitle2.copyWith(
-        color: context.colorScheme.textDisabled,
-      ),
-      borderColor: BorderColor(
-        focusColor: Colors.transparent,
-        unFocusColor: Colors.transparent,
-      ),
-      onTapOutside: (event) {
-        FocusManager.instance.primaryFocus?.unfocus();
-      },
-      prefixIcon: Icon(
-        Icons.search,
-        color: context.colorScheme.textDisabled,
-        size: 24,
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 16.0),
+      child: AppTextField(
+        controller: state.searchController,
+        borderRadius: BorderRadius.circular(30),
+        contentPadding: const EdgeInsets.all(16),
+        borderType: AppTextFieldBorderType.outline,
+        onChanged: (value) => notifier.onSearchChanged(),
+        backgroundColor: context.colorScheme.containerLowOnSurface,
+        hintText: context.l10n.add_team_member_search_placeholder_text,
+        style: AppTextStyle.body2.copyWith(
+          color: context.colorScheme.textPrimary,
+        ),
+        hintStyle: AppTextStyle.subtitle2.copyWith(
+          color: context.colorScheme.textDisabled,
+        ),
+        borderColor: BorderColor(
+          focusColor: Colors.transparent,
+          unFocusColor: Colors.transparent,
+        ),
+        onTapOutside: (event) {
+          FocusManager.instance.primaryFocus?.unfocus();
+        },
+        prefixIcon: Icon(
+          Icons.search,
+          color: context.colorScheme.textDisabled,
+          size: 24,
+        ),
```

Comment:
I think if we need to use same search field in multiple screens so Let's make common searchField.